Cargo.toml

```diff
@@ -34,6 +34,7 @@ tokio-executor = { version = "0.1.0", optional = true }
 tokio-io = "0.1"
 tokio-reactor = { version = "0.1", optional = true }
 tokio-tcp = { version = "0.1", optional = true }
+tokio-threadpool = { version = "0.1", optional = true }
```

Comment:
Since this is only used in the examples, I think this can just go in the `dev-dependencies`, yea?

Reply:
Fixed.  tokio-threadpool is no longer needed.  Added tokio-fs to dev-dependencies.